osu.Game.Rulesets.Mania/Skinning/Legacy/LegacyStageBackground.cs

```diff
@@ -54,7 +54,8 @@ private void load(ISkinSource skin, StageDefinition stageDefinition)
                 },
                 columnBackgrounds = new ColumnFlow<Drawable>(stageDefinition)
                 {
-                    RelativeSizeAxes = Axes.Y
+                    RelativeSizeAxes = Axes.Y,
+                    Masking = false,
```

Comment:
Masking is disabled here because the right line of the last column exceeds the flow's bounds (correctly so).